Formula/python3.rb

```diff
@@ -310,7 +310,22 @@ def post_install
   end
 
   def xy
-    version.to_s.slice(/(3\.\d)/) || "3.6"
```

Comment:
Thanks for all those suggestions. I use the third approach. IMO either parsing or wildcard matching is fragile than direct detection.